DSA-Campaign-Uplift-Estimation/target/appengine-staging/__static__/Compare/script-compare.js

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
```

Comment:
Does the gitignore not work on the unnecessary folders? I see the following repeated files:
DSA-Campaign-Uplift-Estimation/target/appengine-staging/__static__/Compare/script-compare.js
DSA-Campaign-Uplift-Estimation/target/appengine-staging/Compare/script-compare.js
DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js
DSA-Campaign-Uplift-Estimation/target/dsa-uplift-estimation-2020-1/Compare/script-compare.js

Reply:
So we split up the build errors among ourselves - I took the datastore error, Neel took the build trigger error, and Erik is in charge of gitignore. He did create a gitignore file, but I don't think his solution works. We will bring this up during the pod sync.

Reply:
Oh ok, thanks for updating me on that. We should get that working since the code reviews are difficult with so many repeated files. But for now I'll LG.